docs/core-concepts/versioning.md

```diff
@@ -1,6 +1,201 @@
+
+Metapackages
+-------------
+
+Versioning for .NET Core metapackages is based on the target framework that they map to. The metapackages adopt the version number of the highest version (for example, netstandard1.5) of the target framework it maps to in its package closure. The patch version for the metapackage is used to represent updates to the metapackage to reference updated packages (that don't include a new target framework version). As a result, the metapackages are not strictly SemVer compliant because their versioning scheme doesn't represent the degree of change in the underlying packages, but only API level. 
+
+There are two primary metapackages for .NET Core.
+
+**NETStandard.Library**
+
+- v1.5 as of .NET Core 1.0.
+- Maps to `netstandard` target framework. 
+- Describes the packages that are considered required for modern app development and that .NET platforms must implement to be considered a ".NET Standard" platform.
+
+**Microsoft.NETCore.App**
+
+- v1.0 as of .NET Core 1.0 (versions intended to match).
+- Maps to `netcoreapp` target framework.
+- Describes the packages in the .NET Core distribution.
+
+Note: It is reasonable to reference the `Microsoft.NETCore.App` metapackage but use the netstandard framework. You will get access to all of the netstandard-compatible assets from those packages, which will be a subset of all the packages but more than are referenced via `NETStandard.Library`.
+
+
+Target Frameworks
+-----------------
+
+Target Frameworks describe API surface area. As new APIs are added, the framework versions. A new version of a target framework comes into existence when a single package adds a folder with that version and NuGet is updated to respect that new version.
+
+There is a two-way relationship between metapackages and frameworks. One can think of a given metapackage "filling out" the framework. Given that .NET Core is delivered via packages, the .NET Core target frameworks don't inherently define any surface area. It is the presence of the framework folders in the .NET Core packages that defines the APIs available for a given target framework (including version).
+
+### netstandard
+
+.NET Standard is a new concept that came out of the .NET Core project, but is logically separate. It is a a new framework that describes the API evolution of .NET for the lower-most APIs. One could consider it a standardized Base Class Library (BCL) for all .NET implementations. .NET Core implements this spec and is ".NET Standard compliant" at a particular version level. The .NET Standard will be updated regularly, based on new APIs being adding in .NET Standard compliant .NET implementations, such as .NET Core, .NET Framework and Mono. The version number of .NET Standard will not match any of the implementations.
```

Comment:
It is a a -> remove double article 
lowermost -> not hyphened
spec -> from Microsoft Writing Style Guide, Always spell out (specification). Don't use spec.